### PR TITLE
Improve test coverage for components

### PR DESCRIPTION
**Pull request by sudharsana-kjl**
Add unit tests for components to increase test coverage

#232 

Added atleast one test for the components I mentioned in the referenced issue. I'll be working on writing more tests for other pages and components and also improve the coverage for existing tests in future.

### CODE REVIEW

File: test/unit/specs/components/share-list.spec.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import ShareList from '@/components/ShareList';
+import render from '../../test-utils/render';
+
+describe('ShareList', () => {
+  it('should render correct contents', () => {
+    const wrapper = render(ShareList);
+    expect(wrapper.find({ name: 'share-list' }).element).toBeDefined();
+  });
+});
```
Comment (brenoferreira) [inline test/unit/specs/components/share-list.spec.js]: We're removing the lists feature (See #242). So you can delete these tests and not add any more tests for that feature :)

Comment (sudharsana-kjl) [reply test/unit/specs/components/share-list.spec.js]: Removed the tests for share-list. Could you please check now?

Comment (brenoferreira) [reply test/unit/specs/components/share-list.spec.js]: Looks good to me. Let me know if you're planning on adding more tests to this PR

Comment (sudharsana-kjl) [reply test/unit/specs/components/share-list.spec.js]: Hey! I'm planning to add a few more tests to components and pages. I shall comment here once done

Comment (sudharsana-kjl) [reply test/unit/specs/components/share-list.spec.js]: @brenoferreira  Could you please check this?

**Pull request merged**